skit_labels/commands.py

```diff
@@ -202,13 +202,13 @@ def processLabelstudioColumns(df_path: str):
         lambda val: json.dumps(json.loads(json.loads(val)), ensure_ascii=False) \
             if json.loads(val) else json.dumps([])
     )
-    try:
-        df["tag"] = df["tag"].apply(lambda val: json.dumps(json.loads(val)[0]["value"]))
-        df = df[df["tag"].apply(lambda val: "choices" in json.loads(val))]
-        df["tag"] = df["tag"].apply(lambda val: json.loads(val)["choices"][0])
```

Comment:
@biswaroop1547 this also confirms that previously we were giving out a string only for the `tag` column.